webassembly/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM gitpod/workspace-full:latest
+
+USER gitpod
+
+RUN bash -cl "cargo install wasm-pack cargo-wasm cargo-generate \
+    && curl \"https://wasmtime.dev/install.sh\" -sSf | bash"
+
+RUN wget "https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-8/wasi-sdk_8.0_amd64.deb" \
+    && sudo dpkg -i wasi-sdk_8.0_amd64.deb
+
+
+RUN bash -cl "git clone \"https://github.com/emscripten-core/emsdk.git\" \
+    && cd emsdk \
+    && ./emsdk install latest \
+    && ./emsdk activate latest \
+    && source ./emsdk_env.sh"
```

Comment:
There is probably no need to wrap these commands in `bash -cl ""`.

Also, please explicitly mention where you're cloning this, in order to remove ambiguity. E.g.

```Dockerfile
RUN git clone https://github.com/emscripten-core/emsdk /tmp/emsdk && \
    # ... && \
    rm -rf /tmp/emsdk
```

or if you want the cloned directory to remain behind, add in in `$HOME` but prefix the intall-dir with a `.`:

```Dockerfile
RUN git clone https://github.com/emscripten-core/emsdk $HOME/.emsdk && \
    # ...
```
RUN

Reply:
Done!